ca/ca.go

```diff
@@ -270,6 +271,14 @@ func NewCertificateAuthorityImpl(
 		return nil, err
 	}
 
+	if config.Backdate == "" {
+		return nil, errors.New("Config must specify a backdate duration.")
```

Comment:
I think we would want this to be handled gracefully and not produce an error so that the `config/ca.json` can be left at the non-present default value and `config-next/ca.json` can be set to a value.